test/fw/ptl/utils/plugins/ptl_test_runner.py

```diff
@@ -828,12 +828,14 @@ def check_hardware_status_and_core_files(self):
                     _msg += sched_priv_path
                     self.logger.warning(_msg)
             for u in PBS_ALL_USERS:
-                user_home_files = du.listdir(hostname=hostname, path=u.home,
-                                             sudo=True, fullpath=False)
-                if fnmatch.filter(user_home_files, "core*"):
-                    _msg = hostname + ": user-" + str(u)
-                    _msg += ": core files found in "
-                    self.logger.warning(_msg + u.home)
+                if isdir(hostname=hostname, path=u.home):
```

Comment:
You need to use DshUtils() object 'du' to use 'isdir' function.

Reply:
@sakshamgarg Now i am not using 'isdir' functiion.